terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/runners.ts

```diff
@@ -472,6 +473,13 @@ export async function tryReuseRunner(
     repoName: runnerParameters.repoName,
     runnerType: runnerParameters.runnerType.runnerTypeName,
   };
+  if (runnerParameters.runnerType.runnerTypeName.includes('linux.4xlarge')) {
+    if ((await getExperimentValue('stresstest_awsfail', 0)) > Math.random() * 100) {
```

Comment:
Since this double `if` is a common pattern needed, consider extracting it into a single function:
```suggestion
  experiment_root="stresstes_awsfail"
  if (shouldRunExperiment(experiment_root, runnerParameters.runnerType.runnerTypeName)) {
```